assets/src/recurse_bundle_plugins.js

```diff
@@ -9,56 +9,35 @@ var path = require("path");
 var logging = require("./logging");
 var _ = require("lodash");
 
-var readBundlePlugin = require('./read_bundle_plugin');
+var readBundlePlugins = require('./read_bundle_plugins');
 
 /**
- * Take an array of directories, recurse over all directories inside them, and look for any files named
```

Comment:
Thanks, I have to say that my JS skills are really bad.. I can read it and delete it... but writing new stuff... ouch :)